### PR TITLE
fix: properly skip over unrecognized CSI sequences.

### LINKED ISSUE #466: Unknown CSI inputs result in garbage key events
When a terminal CSI sequence is not recognized by bubbletea, it is not completely ignored; instead, bubbletea will match the initial escape byte and the character immediately afterwards as a single key with the alt modifier, then interpret the rest as regular key inputs. For example, ESC [ 1003 h  gets parsed as alt+1, then 0, then 0, then 3, then h.

Since the CSI sequences all have a regular structure, they could be parsed and ignored as a single unit.

(The definition is strict: a CSI sequence starts with `ESC [` and terminates with a byte value in the range 0x40 to 0x7E.)

### PR DESCRIPTION
**Pull request by knz**
Fixes #466.

Also includes a removal of now-unnecessary code.

cc @muesli

### CODE REVIEW

File: key.go [hunk 1]
```diff
@@ -469,18 +468,12 @@ var sequences = map[string]Key{
 	"\x1b\x1b[32~": {Type: KeyF18, Alt: true},
 	"\x1b\x1b[33~": {Type: KeyF19, Alt: true},
 	"\x1b\x1b[34~": {Type: KeyF20, Alt: true},
-}
-
-// Hex code mappings.
-var hexes = map[string]Key{
-	"1b0d": {Type: KeyEnter, Alt: true},
-	"1b7f": {Type: KeyBackspace, Alt: true},
```
Comment (muesli) [inline key.go:477]: Just checking in if this was removed on purpose? Since you added a test for the two sequences above, I assume it's working fine tho.

Comment (knz) [reply key.go:477]: Yes as the commit message says this is now unneeded. It became obsolete when we fixed the alt key a few PRs ago.

Comment (meowgorithm) [reply key.go:477]: I’ll add that I’m thrilled to see those gone. Thanks, @knz.

**Comment by muesli** [comment]
Rebased master onto your branch.

**Pull request merged**